docs/sources/installation/centos.md

```diff
@@ -6,153 +6,171 @@ page_keywords: Docker, Docker documentation, requirements, linux, centos, epel,
 
-The above command will not remove images, containers, volumes, or user created
-configuration files on your host. If you wish to delete all images, containers,
-and volumes run the following command:
+3. Make sure your existing packages are up-to-date.
 
-    $ rm -rf /var/lib/docker
+		$ sudo yum update
+		
+2. Download the Docker RPM to the current directory.
+		
+		$ curl -O -sSL http://get.docker.com/docker-engine-1.7.0-0.1.el6.x86_64.rpm
 
-You must delete the user created configuration files manually.
+4. Use `yum` to install the package.
 
-## CentOS-6.5
+		$ sudo yum localinstall --nogpgcheck docker-engine-1.7.0-0.1.el6.x86_64.rpm
 
-### Installation
+4. Start the Docker daemon.
 
-For CentOS-6.5, the Docker package is part of [Extra Packages
-for Enterprise Linux (EPEL)](https://fedoraproject.org/wiki/EPEL) repository,
-a community effort to create and maintain additional packages for the RHEL distribution.
+		$ sudo service docker start
 
-Firstly, you need to ensure you have the EPEL repository enabled. Please
-follow the [EPEL installation instructions](
-https://fedoraproject.org/wiki/EPEL#How_can_I_use_these_extra_packages.3F).
+4. Verify `docker` is installed correctly.
 
-For CentOS-6, there is a package name conflict with a system tray application
-and its executable, so the Docker RPM package was called `docker-io`.
+		$ sudo docker run hello-world
+		Unable to find image 'hello-world:latest' locally
+		latest: Pulling from hello-world
+		a8219747be10: Pull complete 
+		91c95931e552: Already exists 
+		hello-world:latest: The image you are pulling has been verified. Important: image verification is a tech preview feature and should not be relied on to provide security.
+		Digest: sha256:aa03e5d0d5553b4c3473e89c8619cf79df368babd1.7.0cf5daeb82aab55838d
+		Status: Downloaded newer image for hello-world:latest
+		Hello from Docker.
+		This message shows that your installation appears to be working correctly.
 
-To proceed with `docker-io` installation on CentOS-6, you may need to remove the
-`docker` package first.
+		To generate this message, Docker took the following steps:
+		 1. The Docker client contacted the Docker daemon.
+		 2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
+				(Assuming it was not already locally available.)
+		 3. The Docker daemon created a new container from that image which runs the
+				executable that produces the output you are currently reading.
+		 4. The Docker daemon streamed that output to the Docker client, which sent it
+				to your terminal.
 
-    $ sudo yum -y remove docker
+		To try something more ambitious, you can run an Ubuntu container with:
+		 $ docker run -it ubuntu bash
 
-Next, let's install the `docker-io` package which will install Docker on our host.
+		For more examples and ideas, visit:
+		 http://docs.docker.com/userguide/
 
-    $ sudo yum install docker-io
+  This command downloads a test image and runs it in a container.
```

Comment:
Wondering if this is a bit redundant, given the descriptive output of the "hello world" image.

If you want to keep it, and you're going for the `--rm` flag, perhaps something along;

> This command downloads a test image and runs it in a container. The container is automatically removed afterwards because we provided the `--rm` flag when starting the container.